modules/luci-mod-quick_setup/htdocs/luci-static/resources/view/quick_setup/quick_setup.js

```diff
@@ -0,0 +1,337 @@
+'use strict';
+'require view';
+'require dom';
+'require ui';
+'require form';
+'require rpc';
+'require uci';
+'require network';
+
+var formData={
```

Comment:
Please stick to coding style used in `luci` project and use spaces appropriately. E.g.
```
var foo = {
(...)
};
```